web/themes/new_weather_theme/templates/partials/observation-table.html.twig

```diff
@@ -0,0 +1,76 @@
+<table class="usa-table usa-table--borderless observation-table border-top-1px border-base-light width-full">
+    <tbody>
+        {# WIND #}
+        <tr class="wx-wind-speed">
+            <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Wind" | t }}
+            </th>
+            <td>
+                {% include '@new_weather_theme/partials/wind.html.twig' with { wind: content.wind } %}
+            </td>
+        </tr>
+        {# GUSTS #}
+        {%  if content.gusts is not same as (null) %}
+            <tr class="wx-gusts">
+                <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                    {{ "Gusts" | t }}
+                </th>
+                <td>
+                    {{ content.gusts.speed }} {{  "mph" | t }}
+                </td>
+            </tr>
+        {% endif %}
+        {# HUMIDITY #}
+        <tr class="wx-humidity">
+            <th scope="row" class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Humidity" | t }}
+            </th>
+            <td>
+                {% if content.humidity == 0 %}
+                    {{  "N/A" | t}}
+                {% else %}
+                    {{  content.humidity }}%
+                {% endif %}
+            </td>
+        </tr>
+        {# DEWPOINT #}
+        <tr class="wx-dewpoint">
+            <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Dewpoint" | t }}
+            </th>
+            <td>
+                {% if content.dewpoint is same as (null) %}
+                    {{  "N/A" | t}}
+                {% else %}
+                    {{  content.dewpoint }}<span class="font-body-3xs position-absolute margin-top">&deg;F</span>
+                {% endif %}
+            </td>
+        </tr>
+        {# VISIBILITY #}
+        <tr class="wx-visibility">
+            <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Visibility"  | t }}
+            </th>
+            <td>
+                {% if content.visibility is same as (null) %}
+                    {{  "N/A" | t}}
+                {% else %}
+                    {{  content.visibility }} {{ "miles" | t }}
+                {% endif %}
+            </td>
+        </tr>
+        {# PRESSURE #}
+        <tr class="wx-pressure">
+            <th scope="row"  class="font-family-mono text-base font-mono-xs text-uppercase z-400">
+                {{ "Pressure" | t }}
+            </th>
+            <td>
+                {% if content.pressure is same as (null) %}
+                    {{  "N/A" | t}}
+                {% else %}
+                    {{  "@psi in (@mbar mb)" | t({ "@psi": content.pressure.psi, "@mbar": content.pressure.mbar })}}
```

Comment:
for mbar would it be possible to stick a `&nbps;` between the value and the unit? that way on small screens if it wraps to the next line the unit and the number stick together and "mb" isn't on it's short line

Reply:
https://github.com/weather-gov/weather.gov/pull/1285/commits/12281022d4a3b21167568782ba47e359ee8f1fa1